<ui version="4.0" >
 <class>IOptionsDialog</class>
 <widget class="QDialog" name="IOptionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TuxCards Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TabWidget2" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab1" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="TextLabel1_3" >
         <property name="text" >
          <string>&lt;b>Saving&lt;/b></string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="autosave" >
           <property name="text" >
            <string>Autosave all</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="saveMinutes" >
           <property name="maximumSize" >
            <size>
             <width>50</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text" >
            <string>15</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TextLabel2_3" >
           <property name="text" >
            <string>Minutes</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="saveWhenLeaving" >
         <property name="text" >
          <string>Save Data when leaving TuxCards</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="createBackup" >
         <property name="text" >
          <string>Create Backupfile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Q3GroupBox" name="groupBox7" >
         <property name="title" >
          <string>Execute Command after Saving</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLineEdit" name="pCommandLine" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>&lt;b>Look &amp; Feel&lt;/b></string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Q3GroupBox" name="groupBox7_2" >
         <property name="title" >
          <string>Icon Directory</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLineEdit" name="mpIconDirecory" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab4" >
      <attribute name="title" >
       <string>Tree</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>11</number>
       </property>
       <property name="topMargin" >
        <number>11</number>
       </property>
       <property name="rightMargin" >
        <number>11</number>
       </property>
       <property name="bottomMargin" >
        <number>11</number>
       </property>
       <item>
        <widget class="Q3GroupBox" name="GroupBox7_2" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>Font</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>11</number>
          </property>
          <property name="topMargin" >
           <number>11</number>
          </property>
          <property name="rightMargin" >
           <number>11</number>
          </property>
          <property name="bottomMargin" >
           <number>11</number>
          </property>
          <item>
           <widget class="QPushButton" name="treeFontButton" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Change current Font</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel2_5_2" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>current Font:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="treeFontText" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Font-Text</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab5" >
      <attribute name="title" >
       <string>Editor</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="Q3GroupBox" name="GroupBox7" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>Font</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>11</number>
          </property>
          <property name="topMargin" >
           <number>11</number>
          </property>
          <property name="rightMargin" >
           <number>11</number>
          </property>
          <property name="bottomMargin" >
           <number>11</number>
          </property>
          <item>
           <widget class="QPushButton" name="editorFontButton" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Change current Font</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel2_5" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>current Font:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="editorFontText" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Font-Text</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Q3GroupBox" name="GroupBox5" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>Tab Spacing</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>11</number>
          </property>
          <property name="topMargin" >
           <number>11</number>
          </property>
          <property name="rightMargin" >
           <number>11</number>
          </property>
          <property name="bottomMargin" >
           <number>11</number>
          </property>
          <item>
           <widget class="QLabel" name="TextLabel1" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>Set Tabsize to</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tabSize" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>32767</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Q3ButtonGroup" name="ButtonGroup17" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title" >
          <string>WordWrap</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>11</number>
          </property>
          <property name="topMargin" >
           <number>11</number>
          </property>
          <property name="rightMargin" >
           <number>11</number>
          </property>
          <property name="bottomMargin" >
           <number>11</number>
          </property>
          <item>
           <widget class="QRadioButton" name="noWrap" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>no WordWrap</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="widgetWrap" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>wrap at Window-width</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="columnWrap" >
              <property name="font" >
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>wrap at column</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="wrapColumn" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>40</width>
                <height>32767</height>
               </size>
              </property>
              <property name="font" >
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text" >
               <string>45</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpOkButton" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushButton19" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mpOkButton</sender>
   <signal>released()</signal>
   <receiver>IOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PushButton19</sender>
   <signal>released()</signal>
   <receiver>IOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>columnWrap</sender>
   <signal>toggled(bool)</signal>
   <receiver>wrapColumn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autosave</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveMinutes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeFontButton</sender>
   <signal>clicked()</signal>
   <receiver>IOptionsDialog</receiver>
   <slot>changeTreeFont()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editorFontButton</sender>
   <signal>clicked()</signal>
   <receiver>IOptionsDialog</receiver>
   <slot>changeEditFont()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
